src/MixedContentExtractor.php

```diff
@@ -5,27 +5,33 @@
 use Exception;
 use GuzzleHttp\Psr7\Uri;
 use Illuminate\Support\Collection;
+use Psr\Http\Message\UriInterface;
 use Symfony\Component\DomCrawler\Crawler as DomCrawler;
 
 class MixedContentExtractor
 {
-    public static function extract(string $html, string $currentUri): array
+    public static function extract(string $html, UriInterface $currentUri): array
```

Comment:
Changing the signature is a breaking change. Could you make this work without having to change the signature?

Reply:
Undid breaking change.